cfgov/agreements/management/commands/import_agreements.py

```diff
@@ -0,0 +1,54 @@
+import os
+import urllib
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+
+from .util import upload_to_s3, update_agreement, update_issuer, clear_tables
+
+
+class Command(BaseCommand):
+    """
+    imports credit card agreement data from provided csv
+    """
+
+    help = "Upload agreements data from new Quarterly Agreement file path"
+
+    def add_arguments(self, parser):
+        parser.add_argument('-p', '--path', action='store', required=True)
+
+    def handle(self, *args, **options):
+        now = datetime.now()
+        suffix = '%s_%s_' % (now.month, now.year)
+
+        uri_hostname = 'http://files.consumerfinance.gov'
```

Comment:
See comment below about S3 paths, it'd be nice to avoid this duplication (very reasonable to be in another PR if better).

Reply:
Let's do it in another PR